### PR TITLE
PHP tools version update

### PR DESCRIPTION
**Pull request by omar-lmounir**

### CODE REVIEW

File: php/config.yml [hunk 1]
```diff
@@ -14,14 +14,14 @@ test_config: &test_config
 
 build_args: &build_args
   APCU_VERSION: 5.1.22
-  COMPOSER_VERSION: 2.5.4
-  ICONV_VERSION: 1.15-r3
+  COMPOSER_VERSION: 2.5.7
+  ICONV_VERSION: 1.17-r2
```
Comment (mremi) [inline php/config.yml:18]: please revert this version, which leads to the failure

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 Versions
 ========
 
+2023-06-07
```
Comment (mremi) [inline CHANGELOG.md:4]:
```suggestion
2023-06-30
```

Comment (omar-lmounir) [reply CHANGELOG.md]: c est fait

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 Versions
 ========
 
+2023-06-30
+----------
+* PHP: update tools version (composer, PHP CS FIXER, Xdebug, Musl)
```
Comment (mremi) [inline CHANGELOG.md:6]:
```suggestion
* PHP: update tools version (composer, PHP CS Fixer, Xdebug)
```

Comment (omar-lmounir) [reply CHANGELOG.md]: c est fait

**Pull request merged**